deps/slfritchie_eflame-a08518142126f5fc541a3a3c4a04c27f24448bae/src/eflame.app.src

```diff
@@ -0,0 +1,13 @@
+{application, eflame,
```

Comment:
Well, there's an [`eflame`](https://github.com/proger/eflame) repo and forked by [`slfritchie`](https://github.com/slfritchie/eflame). So, since I was using slfritchie, I wanted to make sure folks knew which repo it was pulled from.